crates/bevy_asset/src/io/embedded/mod.rs

```diff
@@ -104,20 +104,48 @@ impl EmbeddedAssetRegistry {
 #[macro_export]
 macro_rules! embedded_path {
     ($path_str: expr) => {{
-        embedded_path!("/src/", $path_str)
+        embedded_path!("src", $path_str)
     }};
 
     ($source_path: expr, $path_str: expr) => {{
         let crate_name = module_path!().split(':').next().unwrap();
```

Comment:
While we're here in #11340, I added some code in case this fails. Perhaps it's too defensive if we're certain `module_path!()` will always have a ':' in it. But here it is for your consideration:

```
        let Some(crate_name) = module_path!().split(':').next() else {
            panic!(
                "Could not find crate name. Expected ':' in module path `{}`",
                module_path!()
            );
        };
```

Reply:
If the `unwrap` somehow managed to panic, it would be a bug in the compiler (`module_path!` is a compiler built-in), not something the user would be able to fix. So I don't think it's worth adding this message.